catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -0,0 +1,193 @@
+// @flow
+// $FlowFixMe (hooks)
+import {Fragment, useState} from 'react';
+import Router from 'next/router';
+import styled from 'styled-components';
+import TextInput from '@weco/common/views/components/TextInput/TextInput';
+import Icon from '@weco/common/views/components/Icon/Icon';
+import theme from '@weco/common/views/themes/default';
+import {classNames, font} from '@weco/common/utils/classnames';
+import {trackEvent, trackEventV2} from '@weco/common/utils/ga';
+import {worksUrl} from '../../services/catalogue/urls';
+
+const workTypes = [
+  { id: 'a', label: 'Books' },
+  { id: 'b', label: 'Manuscripts, Asian' },
+  { id: 'c', label: 'Music' },
+  { id: 'd', label: 'Journals' },
+  { id: 'e', label: 'Maps' },
+  { id: 'f', label: 'E-videos' },
+  { id: 'g', label: 'Videorecordings' },
+  { id: 'h', label: 'Archives and manuscripts' },
+  { id: 'i', label: 'Sound' },
+  { id: 'j', label: 'E-journals' },
+  { id: 'k', label: 'Pictures' },
+  { id: 'l', label: 'Ephemera' },
+  { id: 'm', label: 'CD-Roms' },
+  { id: 'n', label: 'Cinefilm' },
+  { id: 'p', label: 'Mixed materials' },
+  { id: 'q', label: 'Digital images' },
+  { id: 'r', label: '3-D Objects' },
+  { id: 's', label: 'E-sound' },
+  { id: 'u', label: 'Standing order' },
+  { id: 'v', label: 'E-books' },
+  { id: 'w', label: 'Student dissertations' },
+  { id: 'x', label: 'E-manuscripts, Asian' },
+  { id: 'z', label: 'Web sites ' }
+];
+
+type Props = {|
+  initialQuery: string,
+  initialWorkType: string[],
+  initialItemsLocationsLocationType: string[],
+  showFilters: boolean
+|}
+
+const SearchInputWrapper = styled.div`
+  margin-right: ${8 * theme.spacingUnit}px;
```

Comment:
Could we be adding a <ThemeProvider> at the root of the app, so it's available to all components?